### DISCUSSION TITLE
Feedback trainer selection

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#54] [On hold - maybe later]
New trainer selection for trained by user trainings is great. But if you like to enter eg the device passport of a new person who was trained in other companies you can not enter the trainer. The system requires to create or search for an employee. Otherwise it will not take over the briefing. 

Idea:
Either a button "External trainer" with free text or generally selection: select trainer from employee list or enter free text

**Comment by YvesR** [comment]
- You can do enter external trainer already
- You can also add staff member already (if permission granted) and then select it

I don't see the reason to make any change. Or add more details.

**Reply by moritzvolkmar** [reply]
How can I enter an external trainer for a "trained by user" training without adding him as an employee?

**Reply by YvesR** [reply]
> How can I enter an external trainer for a "trained by user" training without adding him as an employee?

In this case it would be a service-contact and has to be select-able. Correct?

**Reply by moritzvolkmar** [reply]
No, it is not a service contact. The trainer has nothing to do with the tentant. But this topic is something that we already do with my.samedis.care. That's why I'm putting the topic on hold for now.

**Comment by moritzvolkmar** [comment]
Current customer feedback on this:

Ein Pfleger der Intensivstation bei uns hat sich folgende Frage gestellt:
 
Letzte Woche ist mir noch eine Frage in den Sinn gekommen: 
 
Wie gehen wir mit Einweisungen neuer Mitarbeiter/Leasingkräfte um, die diese vorab woanders erhalten haben? Herstellereinweisungen können wir ja mit der Freitexteingabe problemlos übertragen, aber wie sieht es mit Anwendereinweisungen aus? Hier kann ich ja schlecht externe als hauseigene Mitarbeiter anlegen!?
 
Haben Sie dafür einen Lösungsvorschlag?

**Reply by YvesR** [reply]
my.samedis.care > Einweisungen dokumentieren > Mit der Einrichtung teilen.
